tests/tasks/test_db.py

```diff
@@ -25,7 +25,7 @@ def test_handle_db_dump(mongo, mp_dump_data):
             {
                 field: value
                 for field, value in mongo_model_to_json(x).items()
-                if field not in ("dataframe", "tensor", "synthetic")
+                if field not in ("dataframe", "tensor", "embedded_tensor", "synthetic")
```

Comment:
Maybe add a constant tuple - FIELDS_NOT_INLCLUDED_IN_MP

Reply:
Makes it more understandable why we are not checking for these fields

Reply:
good idea, added